src/test/java/com/jcabi/jdbc/JdbcSessionITCase.java

```diff
@@ -91,6 +99,93 @@ public void changesTransactionIsolationLevel() throws Exception {
     }
 
     /**
+     * JdbcSession can run a function (stored procedure) with
+     * output parameters.
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void callsFunctionWithOutParam() throws Exception {
+        final DataSource source = JdbcSessionITCase.source();
+        new JdbcSession(source).autocommit(false).sql(
+            "CREATE TABLE IF NOT EXISTS users (name VARCHAR(50))"
+        ).execute().sql("INSERT INTO users (name) VALUES (?)")
+        .set("Jeff Charles").execute().sql(
+            StringUtils.join(
+                "CREATE OR REPLACE FUNCTION fetchUser(username OUT text,",
```

Comment:
@mkordas Yes, since they are SQL views which hold the result of each SELECT, and they will exist at the same time; I didn't want to "step on each other's toes"